account/utils.py

```diff
@@ -131,7 +131,7 @@ def check_password_expired(user):
     except PasswordHistory.DoesNotExist:
         return False
 
-    now = datetime.datetime.now(tz=pytz.UTC)
+    now = datetime.datetime.now()
```

Comment:
I think this should use `django.utils.timezone.now()`

Reply:
Probably user.account.now() since this will return a timezone aware date relatvie to the users timezone.
@Yogeshddjango - can you make this change?